docker-compose.selenium.yml

```diff
@@ -0,0 +1,162 @@
+version: '3'
+
+services:
```

Comment:
Should this be its own file, or should we just incorporate this into a docker-compose-dev?